os/os-windows.h

```diff
@@ -219,8 +219,8 @@ static inline int fio_mkdir(const char *path, mode_t mode) {
 	return 0;
 }
 
-#define FIO_HAVE_CPU_ONLINE_SYSCONF
-unsigned int cpus_online(void);
+#define FIO_HAVE_CPU_CONF_SYSCONF
```

Comment:
It seems like we could just get rid of this completely and use sysconf() from `os/windows/posix.c`